lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -2880,6 +2885,7 @@ data MkApiTransactionParams = MkApiTransactionParams
     , txMeta :: W.TxMeta
     , txMetadata :: Maybe W.TxMetadata
     , txTime :: UTCTime
+    , txIsValidScript :: Maybe Bool
```

Comment:
I need to choose a more descriptive type for this:

Nothing -> Not an era that cares about validation
Just False -> Cares about validation, but validation failed
Just True -> Cares about validation and validation succeeded

Reply:
99f37ce